src/ContextTypes/Person.php

```diff
@@ -10,6 +10,11 @@ class Person extends AbstractContext
      * @var array
      */
     protected $structure = [
-        'name' => null,
```

Comment:
tests are not happy as 'name' should exist according to "Properties from Thing" (cf. http://schema.org/Person)